src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -18,4 +18,10 @@
     public static final Prefix PREFIX_ITEM_DESCRIPTION = new Prefix("-d ");
     public static final Prefix PREFIX_ITEM_LOCATION = new Prefix("-l ");
 
+    // recipes start here
+    public static final Prefix PREFIX_RECIPE_PRODUCT_NAME = new Prefix("-n ");
+    public static final Prefix PREFIX_RECIPE_INGREDIENTS = new Prefix("-items ");
```

Comment:
can consider shorter prefix

Reply:
This was specified in user guide, can be changed easily regardless